### ISSUE TITLE
How can i contact you ?

### ISSUE THREAD

**Issue by loneae** [stdcoutzyx/DeepID_FaceClassify#3]
I have a very large dataset of 2million identities with 40 million pics, for each identity i have 20 pics.

I have 4 K80 Servers for the train, how can i contact you ?

**Comment by stdcoutzyx** [comment]
Hi, @loneae , sorry for no response for such a long time. I am running for my job after graduation. If u still want to contact me. just send email to me. My email is stdcoutzyx@163.com

**Comment by PierreHao** [comment]
I have wrote my program based on Caffe, with caffe the code will become more brief, and i have tested DeepID and other type of CNN, but in my test, deepid gets only 90% verification accuracy on lfw, while another type of inception cnn gets about 95% verification accuracy on lfw.  I have graduated from BUAA (The same to yours?). I want to make friends with you for other discussion on deep learning in the future. Can I send my QQ or WeChat to you? @stdcoutzyx

**Comment by stdcoutzyx** [comment]
@PierreHao yeah, i am studying in BUAA now, of course you can send your QQ or webChat to me.

**Comment by PierreHao** [comment]
I have sent an email to stdcoutzyx@163.com
